sass/components/_gallery.scss

```diff
@@ -223,23 +223,31 @@ Style guide: gallery.overlay
     height: 100%;
     width: 10%;
 
-    a {
-      cursor: pointer;
-      font-size: 1.4em;
+    .gallery-overlay__media-previous__arrow,
```

Comment:
It seems we don't need this element which would be a little bit redundant. You can use the parent wrapper to identity the selector.